src/gfx/qt.cpp

```diff
@@ -395,7 +395,7 @@ GfxProcQT::GfxProcQT()
     dir.setFilter(QDir::Files);
     foreach(QString dirFile, dir.entryList())
     {
-        LOG_warn << "Removing unexpected temporary file found from previous executions: " << dirFile;
+        LOG_warn << "Removing unexpected temporary file found from previous executions: " << dirFile.toStdString();
```

Comment:
I think we'd rather use `.toUtf8().constData()` : toStdString return encodes in UTF-8 in qt 5, but as utf8 in qt4

Reply:
updated with that change, thanks!